### PR TITLE
Adding a test to the most_popular_cities () function

### PR DESCRIPTION
**Pull request by AnnaLangner**
Adding a test to the most_popular_cities () function

### CODE REVIEW

File: script/main_test.py [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+  except sqlite3.Error as e:
+    print(e)
+  
+  return conn
+
+
+class TestMostPopularCities(unittest.TestCase):
+
+  def test_most_popular_cities(self):
+    self.assertEqual(len(main.most_popular_cities(create_connection("../db/pythonsqliteusers.db"), 5)), 5)
```
Comment (klangner) [inline script/main_test.py]: Create connection once and then use it.
also use setUp method here.
https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp

**Review by klangner — CHANGES_REQUESTED** [review]

File: script/main_test.py (hunk 1 above)
Comment (AnnaLangner) [reply script/main_test.py]: I don't know what I'm doing wrong...

```
-- class TestMostPopularCities(unittest.TestCase): --

  def setUp(self):
    self.connection = create_connection("../db/pythonsqliteusers.db")

  def test_most_popular_cities(self):
    self.assertEqual(len(main.most_popular_cities(connection, 5)), 5)
```

and I get an error: 
`NameError: name 'connection' is not defined`

Comment (AnnaLangner) [reply script/main_test.py]: I solved it ... I forgot the word "self"

File: script/main_test.py [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+    conn = sqlite3.connect(db_file)
+  except sqlite3.Error as e:
+    print(e)
+  
+  return conn
+
+
+class TestMostPopularCities(unittest.TestCase):
+
+  def setUp(self):
```
Comment (klangner) [inline script/main_test.py:18]: Very good.

Please also add function `tearDown()` to close this connection.
https://docs.python.org/3/library/unittest.html#unittest.TestCase.tearDown

**Review by klangner — CHANGES_REQUESTED** [review]

**Review by klangner — APPROVED** [review]

**Pull request merged**